frontend/pulse3d/src/components/interactiveAnalysis/InteractiveAnalysisModal.js

```diff
@@ -121,9 +121,9 @@ const constantModalLabels = {
     buttons: ["Close"],
   },
   duplicate: {
-    header: "Before Running Analysis!",
-    messages: ["Check no double peaks or valleys exist.", "They are shown as red triangles above."],
-    buttons: ["Close"],
+    header: "Warning!",
+    messages: ["Duplicate peaks and valleys detected."],
+    buttons: ["Continue", "Cancel"],
```

Comment:
nit-picky, but all the modals through the desktop app and dashboard have 'cancel' or 'close' on the left. Probably just want to keep them the same in case people do it out of habit